javaparser-core-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/LexicalPreservingPrinterTest.java

```diff
@@ -390,7 +418,51 @@ void printASimpleMethodAddingAStatement() {
                 "    10 + 2;" + EOL +
                 "}", LexicalPreservingPrinter.print(m));
     }
-
+    
+    @Test
+    void printASimpleMethodRemovingAStatement() {
+    	String code = "class A {" 						+ EOL
+    			+ "\t"		+  "foo(int a, int b) {"	+ EOL
+    			+ "\t\t" 	+ "int result = a * b;"		+ EOL
+    			+ "\t\t" 	+ "return a * b;"			+ EOL
+    			+ "\t"		+ "}"						+ EOL
+    			+ "}";
+    			
+    	
+    	CompilationUnit cu = parse(code);
+    	LexicalPreservingPrinter.setup(cu);    	
+    	ExpressionStmt stmt = cu.findAll(ExpressionStmt.class).get(0);
+    	stmt.remove();
+    	
+        assertEqualsNoEol("class A {" 					+ EOL
+    			+ "\t"		+  "foo(int a, int b) {"	+ EOL
```

Comment:
I think (can't check right now) that assertEqualsNoEol matches any EOL in "actual" with "\n" in "expected" - so the left argument should have \n, not EOL.

Reply:
It replaces all EOL characters in "expected" (left argument) with `System.getProperty("line.separator");`. I think the confusion comes from my unnecessary use of the function when the EOL constant is already used (we could also just use `assertEquals` here). I have also just noticed that this test is not ideal because it either passes or fails depending on the system. I could change the EOLs in this test to "\r\n" (both in expected and in actual) to demonstrate the different behavior to the unit test that follows. Would that also make sense from your point of view?

Reply:
Yes, that seems fine, since that's the combination that's giving the trouble here.